strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/CronJobDuplicationStrategy.java

```diff
@@ -0,0 +1,15 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import org.carlspring.strongbox.cron.domain.CronTaskConfigurationDto;
+
+import java.util.Collection;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public interface CronJobDuplicationStrategy
```

Comment:
Also, won't it be better to make the `CronTaskConfigurationDto` just `Comparable`?